fem/gridfunc.hpp

```diff
@@ -462,10 +462,10 @@ class GridFunction : public Vector
 
    /// Returns the Face Jumps error for L2 elements, with 1/h scaling.
    MFEM_DEPRECATED
-   virtual double ComputeDGFaceJumpError(Coefficient *exsol,
-                                         Coefficient *ell_coeff,
-                                         double Nu,
-                                         const IntegrationRule *irs[] = NULL)
+   double ComputeDGFaceJumpError(Coefficient *exsol,
+                                 Coefficient *ell_coeff,
+                                 double Nu,
+                                 const IntegrationRule *irs[] = NULL)
    const;
```

Comment:
```suggestion
                                 const IntegrationRule *irs[] = NULL) const;
```
Tiny nitpick, I know…